index.html

```diff
@@ -1,14 +1,206 @@
 <!DOCTYPE HTML>
 <html>
-  <head>
-    <title>Mockup Home</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
-</html>
\ No newline at end of file
+
+<head>
+  <link href='./style.css' rel='stylesheet' />
+  <title>DEAD ZONE</title>
+  <meta charset="utf-8" />
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Aladin&family=Rock+Salt&display=swap');
+    @import url('https://fonts.googleapis.com/css2?family=BenchNine:wght@300&family=Pathway+Gothic+One&display=swap');
+  </style>
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+</head>
+
+<body>
+
+  <!--- start main grid!-->
+
+  <div class="container">
+    <div class="sidebar-blank1"></div>
+    <a id="blog-zone" href="./index.html">
+
+      <!--- start header/sub header pictures -->
+
+      <div class="header">
+        <p><br><br><br><br>
+        <h2>SCARLET BEGONIAS</h2>
+        <p id="blues">& A TOUCH OF THE BLUES</p><br><br>
+        <br><br><Br></p>
+    </a>
+  </div>
+
+  <div class="sub-header">

```

Comment:
These class names might be better off to be something more specific, maybe something having to do what page your on. rather then doing "container" you could try something like "mainContainer" or "indexContainer"

This would also make it more specific for your CSS file and be sure that you aren't variable shadowing among other files